config/config.go

```diff
@@ -69,7 +69,7 @@ type Config struct {
 	Whitelists      []string `long:"whitelist" description:"Add an IP network or IP that will not be banned. (eg. 192.168.1.0/24 or ::1)"`
 	whitelists      []*net.IPNet
 	//P2P - server ban
-	DisableBanning bool          `long:"nobanning" description:"Disable banning of misbehaving peers"`
+	Banning        bool          `long:"banning" description:"Disable banning of misbehaving peers"`
```

Comment:
Please also fix the description, `Disable` to `Enable`

Reply:
https://github.com/Qitmeer/qitmeer/pull/255/commits/8afc48f6485806fe3dd25d8d664873a01c6da43f